.github/workflows/deep-source.yml

```
@@ -0,0 +1,40 @@
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ github.event.pull_request.head.sha }}
+
+            - name: Install DeepSource scanner
+              run: curl https://deepsource.io/cli | sh
+
+            - name: Set up Node.js 22
+              uses: actions/setup-node@v4
+              with:
+                node-version: 22.x
```

yamllint (1.35.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)